backend/src/test/java/codezap/member/service/MemberServiceTest.java

```diff
@@ -72,36 +90,100 @@ void assertUniquename() {
             Member member = memberRepository.save(MemberFixture.memberFixture());
+            String memberName = member.getName();
 
-            assertThatThrownBy(() -> memberService.assertUniqueName(member.getName()))
+            assertThatThrownBy(() -> memberService.assertUniqueName(memberName))
                     .isInstanceOf(CodeZapException.class)
                     .hasMessage("아이디가 이미 존재합니다.");
         }
     }
 
     @Nested
     @DisplayName("회원 조회 테스트")
-    class findMember {
+    class FindMember {
 
         @Test
         @DisplayName("회원 정보 조회 성공")
         void findMember() {
             Member member = memberRepository.save(MemberFixture.memberFixture());
 
-            assertThat(memberService.findMember(MemberDto.from(member), member.getId()))
-                    .isEqualTo(FindMemberResponse.from(member));
+            FindMemberResponse actual = memberService.findMember(MemberDto.from(member), member.getId());
+
+            assertThat(actual).isEqualTo(FindMemberResponse.from(member));
         }
 
         @Test
         @DisplayName("회원 정보 조회 실패: 본인 정보가 아닌 경우")
         void findMember_Throw() {
             Member member = memberRepository.save(MemberFixture.memberFixture());
+            MemberDto memberDto = MemberDto.from(member);
+            Long otherId = member.getId() + 1;
 
-            assertThatThrownBy(() -> memberService.findMember(MemberDto.from(member), member.getId() + 1))
+            assertThatThrownBy(() -> memberService.findMember(memberDto, otherId))
                     .isInstanceOf(CodeZapException.class)
                     .hasMessage("본인의 정보만 조회할 수 있습니다.");
         }
+
+        @Test
+        @DisplayName("회원 정보 조회 실패: DB에 없는 멤버인 경우")
+        void findMember_Throw_Not_Exists() {
+            Member member = MemberFixture.memberFixture();
+            MemberDto memberDto = MemberDto.from(member);
+            Long memberId = member.getId();
+
+            assertThatThrownBy(() -> memberService.findMember(memberDto, memberId))
+                    .isInstanceOf(CodeZapException.class)
+                    .hasMessage("식별자 " + memberId + "에 해당하는 멤버가 존재하지 않습니다.");
+        }
+    }
+
+    @Nested
+    @DisplayName("템플릿을 소유한 멤버 조회")
+    class GetByTemplateId {
+        @Test
+        @DisplayName("템플릿을 소유한 멤버 조회 성공")
+        void getByTemplateId() {
+            Member member = memberRepository.save(MemberFixture.memberFixture());
+            Category category = categoryRepository.save(CategoryFixture.getFirstCategory());
+            Template template = templateRepository.save(TemplateFixture.get(member, category));
+
+            Member actual = memberService.getByTemplateId(template.getId());
+
+            assertThat(actual).isEqualTo(member);
+        }
+
+        @Test
+        @DisplayName("템플릿을 소유한 멤버 조회 실패 : DB에 없는 템플릿인 경우")
+        void getByTemplateId_Fail() {
+            assertThatCode(() -> memberService.getByTemplateId(100L))
```

Comment:
```suggestion
            Long notSavedId = 100L;
            
            assertThatCode(() -> memberService.getByTemplateId(notSavedId))
```

사소하지만 확인하고 싶은 id도 별도의 변수로 빠져있으면 좋을 것 같습니다.